gems/aws-sdk-core/lib/seahorse/client/plugins/net_http.rb

```diff
@@ -21,7 +21,10 @@ class NetHttp < Plugin
 
         option(:ssl_verify_peer, default: true, doc_type: 'Boolean', docstring: '')
 
-        option(:ssl_ca_bundle, default: nil, doc_type: String, docstring: '')
+        option(:ssl_ca_bundle, doc_type: String, docstring: '') do |cfg|
```

Comment:
Does this need to be updated in the h2 plugin as well?

Reply:
Probably, yeah, i'll do that.